src/containers/InstanceView.tsx

```diff
@@ -22,10 +22,18 @@ function mapStateToProps({ catalog }: IStoreState, { match: { params } }: IRoute
     i => i.metadata.name === params.instanceName && i.metadata.namespace === params.namespace,
   );
   const svcClass = instance
-    ? catalog.classes.find(c => c.metadata.name === instance.spec.clusterServiceClassRef.name)
+    ? catalog.classes.find(
+        c =>
+          !!instance.spec.clusterServiceClassRef &&
```

Comment:
don't think you need the `!!` here as undefined will be falsey

Reply:
need to coerce it to a boolean type because the typedef for `.find()` expects a boolean return value

Reply:
:+1: